pages/AdListPage/[id]/ContactPage/index.js

```diff
@@ -0,0 +1,83 @@
+import useSWR from "swr";
+import { fetcher } from "../../../../helpers/api";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function ContactPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const { data: ad, error } = useSWR(`/api/ads/${id}`, fetcher);
+
+  if (error) return <h1>...sorry cannot contactdata</h1>;
+
+  if (!ad) return <h1>...please wait while loading...</h1>;
+
+  return (
+    <StyledArticle>
+      <UserContainer>
+        <UserProfilPhoto
+          src={
+            ad.userPictureSrc == ""
+              ? `https://source.unsplash.com/random/?person${ad.userName}`
+              : ad.userPictureSrc
+          }
+          width={40}
+          height={40}
+          alt={`Profilphoto of ${ad.userName}`}
+        />
+        <h2>{ad.userName}´s Ad</h2>
+      </UserContainer>
+      <h3>You can reach your Hero</h3>
+      <p>via mail</p>
+      <a href={`mailto:${ad.userEmail}`}>{ad.userEmail}</a>
+      <p>via phone</p>
+      <a href={`tel:${ad.userPhonenumber}`}>{ad.userPhonenumber}</a>
+      <StyledButton
+        type="button"
+        onClick={() => router.push(`/AdListPage/${id}/`)}
+      >
+        go back
+      </StyledButton>
```

Comment:
As you are redirecting the user here to another page, it would be best to use the `Link` component instead of a `button`.